lib/reversetunnel/agent.go

```diff
@@ -595,6 +630,60 @@ func (a *agent) handleChannels() error {
 	}
 }
 
+// sendKeepalives sends keepalive requests to the server at regular intervals configured by [agent.keepAlive].
+//
+// It's possible on older servers that the keepalive request is not supported, the SendRequest will block until until
+// connection is destroyed. If the server responds successfully and the watchdog is configured, the watchdog will then
+// be enabled.
+func (a *agent) sendKeepalives() error {
+	first := true
+	ticker := time.NewTimer(0)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-a.ctx.Done():
+			return nil
+
+		case <-ticker.C:
+			ticker.Reset(retryutils.SeventhJitter(a.keepAlive))
+			const wantReply = true
+			_, _, err := a.client.SendRequest(a.ctx, teleport.KeepAliveReqType, wantReply, nil)
```

Comment:
nit, personal preference

```suggestion
			const wantReplyTrue = true
			_, _, err := a.client.SendRequest(a.ctx, teleport.KeepAliveReqType, wantReplyTrue, nil)
```

Reply:
ack, done in a43f03431becd82ba5ed2f59e584fdf5ecc34df2